packages/proto-signing/src/signing.ts

```diff
@@ -34,13 +34,15 @@ export function makeAuthInfoBytes(
   signers: ReadonlyArray<{ readonly pubkey: Any; readonly sequence: number }>,
   feeAmount: readonly Coin[],
   gasLimit: number,
+  feePayer: string,
```

Comment:
Let's give this some proper optional type: `string | undefined`

Reply:
Sure, I will incorporate this change.